src/components/Breadcrumb/Breadcrumb.story.jsx

```diff
@@ -1 +1,30 @@
-export default from 'carbon-components-react/lib/components/Breadcrumb/Breadcrumb-story';
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { BreadcrumbItem as CarbonBreadcrumbItem } from 'carbon-components-react';
+
+import Breadcrumb from './Breadcrumb';
+
+const BreadcrumbProps = {
+  onClick: action('click'),
+};
+
+storiesOf('Watson IoT|Breadcrumb', module)
```

Comment:
The stories here need to include all the same [stories provided by the Carbon storybook](https://angular.carbondesignsystem.com/?path=/story/components-breadcrumb--basic), in addition to the ones showing extra functionality provided by this PR